core/include/traccc/ambiguity_resolution/resolution_config.hpp

```diff
@@ -0,0 +1,28 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2025 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// System include(s).
+#include <cstdint>
+
+namespace traccc {
+
+enum class resolution_status : uint32_t { UNKNOWN, REJECT, ACCEPT, MAX_STATUS };
```

Comment:
Is this type meant to be given back to the user? (Currently it's only used inside the algorithm, with the user not seeing it.)

If it's an internal type, you might as well move its declaration into the `.cpp` file. 🤔

Reply:
Moved to .cpp now